build/downloader/src/cli.ts

```diff
@@ -30,42 +30,43 @@ export const initProject = async (packageLocation: string, common: CommonArgs) =
 
   const packageJson = await fse.readJson(packageLocation)
   for (const toolName of Object.keys(toolsList)) {
-    let forceDownload = false
-    let toolVersion = packageJson[toolName]?.version || packageJson[toolName]
-    const devBranch = packageJson[toolName]?.devBranch
-
-    if (!toolVersion) {
+    if (!packageJson[toolName]) {
       logger.info(`Version missing for tool ${toolName} in package.json`)
       continue
     }
 
-    const releases = await getReleasedFiles(toolName, common.platform)
-    const devRelease = devBranch && releases.find(x => x.version.endsWith(devBranch))
+    let forceDownload = false
+    let binaryInfo: ProcessedRelease | undefined
+    const { version, devBranch } = packageJson[toolName]
+
+    const releases = await getBinaries(toolName, common.platform, devBranch)
+    const devRelease = devBranch && releases.find(x => x.version.endsWith(sanitizeBranch(devBranch)))
 
     if (devBranch && devRelease && !yn(process.env.IGNORE_DEV_BRANCH)) {
```

Comment:
I know it's not directly related to this PR, but personally, I find `IGNORE_DEV_BRANCH` really confusing! Wouldn't prefixing the env var with something like `CLI_` or `DOWNLOADER_` be more telling?